Classes/Service/ControllerContextFactory.php

```diff
@@ -48,4 +42,23 @@ public function buildControllerContext(UriInterface $uri): ControllerContext
         );
     }
 
+    private static function createHttpRequestFromGlobals(UriInterface $uri): ServerRequest
+    {
+        $_SERVER['FLOW_REWRITEURLS'] = '1';
+        $fromGlobals = ServerRequest::fromGlobals();
+
+        return new ServerRequest(
+            $fromGlobals->getMethod(),
+            $uri,
+            $fromGlobals->getHeaders(),
+            $fromGlobals->getBody(),
+            $fromGlobals->getProtocolVersion(),
+            array_merge(
+                $fromGlobals->getServerParams(),
+                // Empty SCRIPT_NAME to prevent "./flow" in Uri
+                ['SCRIPT_NAME' => '']
```

Comment:
See https://github.com/neos/flow/blob/b8cdaefb705ff8cda1d6d4d17ee7f1c70d4e200d/Classes/Mvc/Routing/UriBuilder.php#L363 and https://github.com/neos/flow/blob/006baf1965ce3dad9baddf251e540de4aff3478e/Classes/Http/Helper/RequestInformationHelper.php#L35